<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>801</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Main Menu</string>
     </attribute>
     <widget class="QLabel" name="label1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>260</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Welcome to the Main Menu! 
 Register if it's your first time using this app.</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>250</width>
        <height>150</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="etUser">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>30</y>
         <width>113</width>
         <height>24</height>
        </rect>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Username</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="etPass">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>60</y>
         <width>113</width>
         <height>24</height>
        </rect>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnRegister">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>100</y>
         <width>80</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Register</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnLogin">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>100</y>
         <width>80</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="btnExit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>290</y>
        <width>170</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>200</y>
        <width>250</width>
        <height>100</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="btnInfo">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>10</y>
         <width>130</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Add Your Information</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnMarket">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>50</y>
         <width>130</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Go To The Market!</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>250</width>
        <height>150</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="etName">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>11</y>
         <width>230</width>
         <height>24</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>40</number>
       </property>
       <property name="placeholderText">
        <string>First and Last name</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="etId">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>42</y>
         <width>130</width>
         <height>24</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>10</number>
       </property>
       <property name="placeholderText">
        <string>National ID Number</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="etShaba">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>98</y>
         <width>130</width>
         <height>24</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>10</number>
       </property>
       <property name="placeholderText">
        <string>Shaba Number</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="etAccNum">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>130</width>
         <height>24</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>10</number>
       </property>
       <property name="placeholderText">
        <string>Account Number</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>IR</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Market</string>
     </attribute>
     <widget class="QLabel" name="label2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>250</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Welcome to The Market!</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>150</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Go to The Main Menu</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionMain">
   <property name="text">
    <string>Main</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
